src/trace-writer.ts

```diff
@@ -163,8 +165,11 @@ export class TraceWriter extends common.Service {
             labels[TraceLabels.GAE_VERSION] = versionLabel;
           }
         }
-        Object.freeze(labels);
-        this.defaultLabels = labels;
+        // Coerce values to strings.
+        for (const key of Object.keys(labels)) {
+          this.defaultLabels[key] = `${labels[key]}`;
```

Comment:
Why not just convert each value to a string when it is filled into `labels`?

Reply:
I could do that. Do you think `${x}` is an adequate way of doing so?

Reply:
I think so because `${x}` is translated to `x.toString()`, I think.

Reply:
Ok, sounds good. Please see the PR diff: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/713/files

I've updated it so that fields that aren't certain to be strings are now coerced to strings. That is, user-config service version and instanceID.